policies/legacy-migration.md

```diff
@@ -0,0 +1,42 @@
+# Legacy Provider Policy
+
+## Purpose
+The Legacy Provider exists to create an opt-in availability mechanism for
+algorithms that, for various reasons, should have their use discouraged.  These
+reasons include, but are not limited to:
+
+* Discovered security issues leaving the algorithm in question unsafe for
+  general use
+
+* Lack of popular use (i.e. balancing code size vs consumption frequency)
+
+OpenSSL recognizes that consumption of these algorithms may continue to be
+required by consuming applications after the conditions above have been
+recognized.  The Legacy provider exists to provide a mechanism for such
+applications to continue having access to these algorithms while preventing
+applications that don't require them from inadvertently using them.
+
+## Constraints on moving an algorithm to the legacy provider
+
+1) Migration of an algorithm to the legacy provider must occur on a semantically
+versioned major release boundary.  Once a major release includes a given
+algorithm in a given provider, it must remain there for every minor release in
+that major stream.
+
+2) Prior to migration, the migration must be announced for at least 1
```

Comment:
Nit: I've been taught that "at least one" is better than "at least 1" in running text like this.